src/Engine/TranskribusClient.php

```
@@ -0,0 +1,179 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Engine;
+
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TranskribusClient
+{
+
+    /** @var HttpClientInterface */
+    private $httpClient;
+
+    /** @var int transkribus process ID. */
+    private $processId;
+
+    /** @var string transkribus access token. */
+    private $accessToken = '';
+
+    /** @var string Error message. */
+    private $errorMessage;
+
+     /** @var bool response error status . */
+    private $reponseHasError = false;
+    
+    /** @var string Transcribed result. */
+    private $textResult = '';
+    
+    /** @var $config. */
+    private $config = [];
+
+    /** @var $langs valid languages. */
+    private $langs = '';
+
+    /** @var string Transcribed result. */
+    private const PROCESSES_URL = "https://transkribus.eu/processing/v1/processes";
+    
+    /**
+     * TranskribusClient constructor.
+     * @param HttpClientInterface $httpClient
+     * @param string $accessToken
+     */
+    public function __construct(
+        HttpClientInterface $httpClient,
+        string $accessToken
+    ) {
+        $this->httpClient = $httpClient;
+        $this->accessToken = $accessToken;
+    }
+
+    /**
+     * @param string $imageURL
+     */
+    public function initProcess(string $imageURL): self
```

Why is this a fluent method? (i.e. returns itself). I would suggest that this should either throw an exception in case of an error, or return the process ID (or whatever else is required).